### PR TITLE
Milestone #1: Getting familiar with Linters for codebase validation

### PR DESCRIPTION
**Pull request by mwenyoa**
During this milestone I learnt something about HTML and CSS but mostly importantly **Linters** which are tools that force me to follow the good codebase structure to meet the standard related to HTML and CSS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,45 @@
...
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
```
Comment (OLIPLICHE) [inline README.md]: - Please remove unused sections from your README to look more professional.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,45 @@
...
-
-### Deployment
-
-
+- Run [npx hintrc .] to test the markup
+- Run [npx stylelint "**/**.{css,scss}] to test the stylesheet
 
 ## Authors
 
 👤 **Author1**
```
Comment (OLIPLICHE) [inline README.md]: - Kindly add your full name and remove `Author1`

File: README.md [hunk 3]
```diff
@@ -1,56 +1,45 @@
...
-👤 **Author2**
+- GitHub: [@mwenyoa](https://github.com/mwenyoa)
+- Twitter: [@anthony_mwenyo](https://twitter.com/anthony_mwenyo)
+- LinkedIn: [@anthony-mwenyo](https://linkedin.com/in/anthony-mwenyo)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 ## 🤝 Contributing
```
Comment (OLIPLICHE) [inline README.md:40]: - Please, add `Show your support`, `Acknowledgments`, `Licence` sections in your ReadMe file. kindly Open the ReadMe [Template](https://github.com/microverseinc/readme-template) for more details 👍

File: README.md [hunk 4]
```diff
@@ -1,56 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
```
Comment (OLIPLICHE) [inline README.md]: Please replace `Project Name` with `Hello-Microverse` .👍

**Review by OLIPLICHE — COMMENTED** [review]
Hi @mwenyoa👋, 

Good job so far!
There are some issues that you still need to work on to get your project approved, but you are almost there!

image



## Good Points:
Great job so far!! 👏 
- Linter is green ✔️ 
-  You added .gitignore file ✔️ 
- You added .html and .css files in separate commit. ✔️

## Required Changes ♻️

- [ ] Kindly make sure the Pull Request have a summary describing introduced changes in more detail
- [ ] Check the comments under the review⬇.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## You can also consider👍:
- Please capitalize your commit messages and use Imperative tense 🥇 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @mwenyoa👋, 

Good job so far!
There are some issues that you still need to work on to get your project approved, but you are almost there!

image



## Good Points:
Great job so far!! 👏 
- Linter is green ✔️ 
-  You added .gitignore file ✔️ 
- You added .html and .css files in separate commit. ✔️

## Required Changes ♻️

- [ ] Kindly make sure the Pull Request have a summary describing introduced changes in more detail
- [ ] Check the comments under the review⬇.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## You can also consider👍:
- Please capitalize your commit messages and use Imperative tense 🥇 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @mwenyoa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**